ircDDBGateway/APRSTransmit/APRSTransmitAppD.cpp

```diff
@@ -79,8 +79,15 @@ int main(int argc, char** argv)
 	}
 
 	wxString repeater = parser.GetParam(0U);
+	if (repeater.length() != LONG_CALLSIGN_LENGTH) {
+		wxString callErrorMsg;
+		callErrorMsg << wxT("Invalid repeater call. ") << repeater << wxT("Call must be ") << LONG_CALLSIGN_LENGTH << wxT(" characters long.\n");
```

Comment:
I have the same understanding. However I think using wxS shall be the way to go as it saves some CPU since conversion is made at compile time rather than runtime.